js/searchbar/hostsSuggestionsPlugin.js

```diff
@@ -0,0 +1,45 @@
+var hosts = [];
+require('hostile').get(false, function (err, lines) {
+    if (!err) {
+        lines.forEach(function (line) {
+            line.forEach(function (host) {
+                host = host.split(' ');
+                host.forEach(function (host) {
+                    if (hosts.indexOf(host) === -1) {
+                        hosts.push(host);
+                    }
+                });
+            })
+        });
+    }
+});
+
+function showHostsSuggestions(text, input, event, container) {
+
+    empty(container);
+
+    var results = hosts.filter(function (host) {
+        return host.indexOf(text) > -1;
+    });
+
+    results.slice(0, 4).forEach(function (result) {
+
+        var item = createSearchbarItem({
+            title: result,
+            secondaryText: 'via hosts file',
```

Comment:
This should probably be a noun, to match the other results (like "suggested site"). "Hosts file entry" maybe?